wasm.md

```diff
@@ -21,11 +21,14 @@ Configuration
       </curFrame>
       <nextFreshId> 0 </nextFreshId>
       <moduleInst>
-        <funcIds> .Map </funcIds> //this is mapping from identifier to index
-        <nextFuncIdx>   0 </nextFuncIdx>
-        <nextTabIdx>    0 </nextTabIdx>
-        <nextMemIdx>    0 </nextMemIdx>
-        <nextGlobalIdx> 0 </nextGlobalIdx>
+        <typeIds>       .Map </typeIds>
+        <funcIds>       .Map </funcIds> //this is mapping from identifier to index
+        <nextTypeIdx>   0    </nextTypeIdx>
+        <nextFuncIdx>   0    </nextFuncIdx>
+        <nextTabIdx>    0    </nextTabIdx>
+        <nextMemIdx>    0    </nextMemIdx>
```

Comment:
Hmm, maybe we don't need `nextMemIdx` and `nextTabIdx` until it actually becomes part of the Wasm spec to allow several? I like the principle of not adding things we don't use. It's not an issue on this PR (since it wasn't introduced here), just wanted to raise a flag about it.